code/modules/research/xenoarchaeology/finds/finds.dm

```diff
@@ -296,9 +296,7 @@
 		if(20)
 			//arcane clothing
 			apply_prefix = 0
-			var/list/possible_spawns = list(/obj/item/clothing/head/culthood,
-			/obj/item/clothing/head/magus,
-			/obj/item/clothing/head/culthood/alt,
+			var/list/possible_spawns = list(/obj/item/clothing/head/magus,
```

Comment:
The new cult garments have hoods attached to them, not as separate helmets.

If you find cult robes, you automatically have their hoods, just like in real life. 

#makess13realisticagain